src/notation/qml/MuseScore/NotationScene/PercussionPanel.qml

```diff
@@ -151,6 +162,15 @@ Item {
 
                 model: percModel.padListModel
 
+                Connections {
+                    target: padGrid.model
+
+                    function onRowIsEmptyChanged(row, isEmpty) {
+                        var showDeleteButton = padGrid.numRows > 1 && isEmpty
```

Comment:
Shouldn't it be showDeleteButton = padGrid.numRows > 1 && **!isEmpty** instead? Also, as far as I understand, you can move this connection inside the delete btn component. That way, we can avoid adding the showDeleteButton alias

Reply:
The button should only be visible if the associated row _is_ empty so that part should be fine - but yeah lets move this to the button component.